packages/storage/amplify_storage_s3_dart/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.3.2
+
+### Fixes
+- fix(smithy): Union variant naming ([#3415](https://github.com/aws-amplify/amplify-flutter/pull/3415))
+
+### Features
+- feat(logging): logger can register one plugin per type in a logger hierarchy ([#3173](https://github.com/aws-amplify/amplify-flutter/pull/3173))
+
```

Comment:
I don't think this should be part of the changelog.

Reply:
Okay, I will remove it. For future reference, anything with scope `feat` is automatically included when generating changelogs.